plugin crashes when modifying rancher project

When trying to run this plan the plugin crashes with a segfault.

These changes happen all the time, because i did not specify `project_monitoring_input ` but it worked without problems before.

happend with terraform `0.14.5`, `0.14.10` and `0.15.2` with plugin version `terraform-provider-rancher2_v1.13.0`


```
❯ terraform plan
Acquiring state lock. This may take a few moments...
vault_policy.projectpolicies["preisschild"]: Refreshing state... [id=subdomain-dev_preisschild-read-policy]
vault_policy.projectpolicies["eec"]: Refreshing state... [id=subdomain-dev_eec-read-policy]
vault_policy.projectpolicies["portal"]: Refreshing state... [id=subdomain-dev_portal-read-policy]
vault_policy.projectpolicies["service"]: Refreshing state... [id=subdomain-dev_service-read-policy]
vault_auth_backend.k8s_auth_backend: Refreshing state... [id=k8s_subdomain-dev]
vault_policy.projectpolicies["demo"]: Refreshing state... [id=subdomain-dev_demo-read-policy]
vault_policy.projectpolicies["expertpos"]: Refreshing state... [id=subdomain-dev_expertpos-read-policy]
vault_policy.projectpolicies["supplychain"]: Refreshing state... [id=subdomain-dev_supplychain-read-policy]
vault_policy.projectpolicies["neo"]: Refreshing state... [id=subdomain-dev_neo-read-policy]
kubernetes_service_account.test-serviceaccount: Refreshing state... [id=default/test-serviceaccount2]
kubernetes_cluster_role_binding.vault-tokenreview-sa-role-binding: Refreshing state... [id=vault-tokenreview-role-binding]
kubernetes_service_account.vault-tokenreview-sa: Refreshing state... [id=default/vault-tokenreview]
vault_kubernetes_auth_backend_role.projectroles["service"]: Refreshing state... [id=auth/k8s_subdomain-dev/role/service]
vault_kubernetes_auth_backend_role.projectroles["eec"]: Refreshing state... [id=auth/k8s_subdomain-dev/role/eec]
vault_kubernetes_auth_backend_role.projectroles["preisschild"]: Refreshing state... [id=auth/k8s_subdomain-dev/role/preisschild]
vault_kubernetes_auth_backend_role.projectroles["neo"]: Refreshing state... [id=auth/k8s_subdomain-dev/role/neo]
vault_kubernetes_auth_backend_role.projectroles["portal"]: Refreshing state... [id=auth/k8s_subdomain-dev/role/portal]
vault_kubernetes_auth_backend_role.projectroles["expertpos"]: Refreshing state... [id=auth/k8s_subdomain-dev/role/expertpos]
vault_kubernetes_auth_backend_role.projectroles["supplychain"]: Refreshing state... [id=auth/k8s_subdomain-dev/role/supplychain]
vault_kubernetes_auth_backend_role.projectroles["demo"]: Refreshing state... [id=auth/k8s_subdomain-dev/role/demo]
rancher2_catalog.jetstack: Refreshing state... [id=jetstack]
vault_kubernetes_auth_backend_config.k8s_auth_backend_config: Refreshing state... [id=auth/k8s_subdomain-dev/config]
rancher2_cluster_role_template_binding.portaltruppe_projectview: Refreshing state... [id=c-g7hnt:portaltruppeprojectview]
rancher2_cluster_role_template_binding.portaltruppe_clustermember: Refreshing state... [id=c-g7hnt:portaltruppeclustermember]
rancher2_project.cert-manager: Refreshing state... [id=c-g7hnt:p-plljx]
rancher2_project.expert-infra: Refreshing state... [id=c-g7hnt:p-b8vcz]
rancher2_project.projekte["eec"]: Refreshing state... [id=c-g7hnt:p-5pzwn]
rancher2_project.projekte["preisschild"]: Refreshing state... [id=c-g7hnt:p-h68l4]
rancher2_project.projekte["demo"]: Refreshing state... [id=c-g7hnt:p-6n5l9]
rancher2_project.projekte["service"]: Refreshing state... [id=c-g7hnt:p-mwkpb]
rancher2_project.projekte["expertpos"]: Refreshing state... [id=c-g7hnt:p-jq4h4]
rancher2_project.projekte["portal"]: Refreshing state... [id=c-g7hnt:p-rnpg9]
rancher2_project.projekte["neo"]: Refreshing state... [id=c-g7hnt:p-6srjh]
rancher2_token.vault-token: Refreshing state... [id=token-9d2pc]
rancher2_secret.azuredns-config: Refreshing state... [id=p-plljx:azuredns-config]
rancher2_namespace.cert-manager: Refreshing state... [id=cert-manager]
rancher2_namespace.istio-gateways: Refreshing state... [id=istio-gateways]
rancher2_namespace.istio-serviceentries: Refreshing state... [id=istio-serviceentries]
rancher2_namespace.memcached: Refreshing state... [id=memcached]
rancher2_namespace.redis: Refreshing state... [id=redis]
rancher2_namespace.mongodb: Refreshing state... [id=mongodb]
rancher2_namespace.rabbitmq: Refreshing state... [id=rabbitmq]
rancher2_namespace.kapp-apps: Refreshing state... [id=kapp-apps]
rancher2_namespace.namespaces["portal"]: Refreshing state... [id=portal]
rancher2_namespace.namespaces["expertpos"]: Refreshing state... [id=expertpos]
rancher2_namespace.namespaces["service"]: Refreshing state... [id=service]
rancher2_namespace.namespaces["demo"]: Refreshing state... [id=demo]
rancher2_namespace.namespaces["preisschild"]: Refreshing state... [id=preisschild]
rancher2_namespace.namespaces["neo"]: Refreshing state... [id=neo]
rancher2_namespace.namespaces["eec"]: Refreshing state... [id=eec]
rancher2_project_role_template_binding.portaltruppe["preisschild"]: Refreshing state... [id=p-h68l4:portalberechtigungen]
rancher2_project_role_template_binding.portaltruppe["portal"]: Refreshing state... [id=p-rnpg9:portalberechtigungen]
rancher2_project_role_template_binding.portaltruppe["demo"]: Refreshing state... [id=p-6n5l9:portalberechtigungen]
rancher2_project_role_template_binding.portaltruppe["neo"]: Refreshing state... [id=p-6srjh:portalberechtigungen]
rancher2_project_role_template_binding.portaltruppe["service"]: Refreshing state... [id=p-mwkpb:portalberechtigungen]
rancher2_project_role_template_binding.portaltruppe["eec"]: Refreshing state... [id=p-5pzwn:portalberechtigungen]
rancher2_project_role_template_binding.portaltruppe["expertpos"]: Refreshing state... [id=p-jq4h4:portalberechtigungen]
rancher2_registry.expertreg["expertpos"]: Refreshing state... [id=p-jq4h4:expertreg]
rancher2_registry.expertreg["eec"]: Refreshing state... [id=p-5pzwn:expertreg]
rancher2_registry.expertreg["portal"]: Refreshing state... [id=p-rnpg9:expertreg]
rancher2_registry.expertreg["preisschild"]: Refreshing state... [id=p-h68l4:expertreg]
rancher2_registry.expertreg["demo"]: Refreshing state... [id=p-6n5l9:expertreg]
rancher2_registry.expertreg["neo"]: Refreshing state... [id=p-6srjh:expertreg]
rancher2_registry.expertreg["service"]: Refreshing state... [id=p-mwkpb:expertreg]
rancher2_app.mongodb-preisschild: Refreshing state... [id=p-b8vcz:mongodb-preisschild]
rancher2_app.cert-manager: Refreshing state... [id=p-plljx:cert-manager]
rancher2_app.redis-session: Refreshing state... [id=p-b8vcz:redis-session]
rancher2_app.redis-neo: Refreshing state... [id=p-b8vcz:redis-neo]
rancher2_app.redis-preisschild: Refreshing state... [id=p-b8vcz:redis-preisschild]
rancher2_app.redis-eec: Refreshing state... [id=p-b8vcz:redis-eec]
rancher2_app.memcached-portal: Refreshing state... [id=p-b8vcz:memcached-portal]
rancher2_app.rabbitmq-preisschild: Refreshing state... [id=p-b8vcz:rabbitmq-preisschild]
kubernetes_service_account.projekt_serviceaccounts["demo"]: Refreshing state... [id=demo/demo]
kubernetes_service_account.projekt_serviceaccounts["expertpos"]: Refreshing state... [id=expertpos/expertpos]
kubernetes_service_account.projekt_serviceaccounts["eec"]: Refreshing state... [id=eec/eec]
kubernetes_service_account.projekt_serviceaccounts["portal"]: Refreshing state... [id=portal/portal]
kubernetes_service_account.projekt_serviceaccounts["preisschild"]: Refreshing state... [id=preisschild/preisschild]
kubernetes_service_account.projekt_serviceaccounts["neo"]: Refreshing state... [id=neo/neo]
kubernetes_service_account.projekt_serviceaccounts["service"]: Refreshing state... [id=service/service]

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create
  ~ update in-place

Terraform will perform the following actions:

  # kubernetes_service_account.projekt_serviceaccounts["supplychain"] will be created
  + resource "kubernetes_service_account" "projekt_serviceaccounts" {
      + automount_service_account_token = true
      + default_secret_name             = (known after apply)
      + id                              = (known after apply)

      + metadata {
          + generation       = (known after apply)
          + name             = "supplychain"
          + namespace        = "supplychain"
          + resource_version = (known after apply)
          + self_link        = (known after apply)
          + uid              = (known after apply)
        }
    }

  # rancher2_namespace.namespaces["supplychain"] will be created
  + resource "rancher2_namespace" "namespaces" {
      + annotations      = (known after apply)
      + description      = "Supply-Chain Projekt"
      + id               = (known after apply)
      + labels           = {
          + "istio-injection" = "enabled"
        }
      + name             = "supplychain"
      + project_id       = (known after apply)
      + wait_for_cluster = false

      + resource_quota {
          + limit {
              + config_maps              = (known after apply)
              + limits_cpu               = (known after apply)
              + limits_memory            = (known after apply)
              + persistent_volume_claims = (known after apply)
              + pods                     = (known after apply)
              + replication_controllers  = (known after apply)
              + requests_cpu             = (known after apply)
              + requests_memory          = (known after apply)
              + requests_storage         = (known after apply)
              + secrets                  = (known after apply)
              + services                 = (known after apply)
              + services_load_balancers  = (known after apply)
              + services_node_ports      = (known after apply)
            }
        }
    }

  # rancher2_project.projekte["neo"] will be updated in-place
  ~ resource "rancher2_project" "projekte" {
        id                        = "c-g7hnt:p-6srjh"
        name                      = "neo"
        # (6 unchanged attributes hidden)

      - project_monitoring_input {
          - answers = {
              - "grafana.persistence.enabled"               = "false"
              - "grafana.persistence.size"                  = "10Gi"
              - "grafana.persistence.storageClass"          = "default"
              - "prometheus.persistence.enabled"            = "false"
              - "prometheus.persistence.size"               = "50Gi"
              - "prometheus.persistence.storageClass"       = "default"
              - "prometheus.resources.core.limits.cpu"      = "1000m"
              - "prometheus.resources.core.limits.memory"   = "4000Mi"
              - "prometheus.resources.core.requests.cpu"    = "750m"
              - "prometheus.resources.core.requests.memory" = "750Mi"
              - "prometheus.retention"                      = "12h"
            } -> null
        }
    }

  # rancher2_project.projekte["portal"] will be updated in-place
  ~ resource "rancher2_project" "projekte" {
        id                        = "c-g7hnt:p-rnpg9"
        name                      = "portal"
        # (6 unchanged attributes hidden)

      - project_monitoring_input {
          - answers = {
              - "grafana.persistence.enabled"               = "false"
              - "grafana.persistence.size"                  = "10Gi"
              - "grafana.persistence.storageClass"          = "default"
              - "prometheus.persistence.enabled"            = "false"
              - "prometheus.persistence.size"               = "50Gi"
              - "prometheus.persistence.storageClass"       = "default"
              - "prometheus.resources.core.limits.cpu"      = "1000m"
              - "prometheus.resources.core.limits.memory"   = "4000Mi"
              - "prometheus.resources.core.requests.cpu"    = "750m"
              - "prometheus.resources.core.requests.memory" = "750Mi"
              - "prometheus.retention"                      = "12h"
            } -> null
        }
    }

  # rancher2_project.projekte["supplychain"] will be created
  + resource "rancher2_project" "projekte" {
      + annotations               = (known after apply)
      + cluster_id                = "c-g7hnt"
      + description               = "Supply-Chain Projekt"
      + enable_project_monitoring = true
      + id                        = (known after apply)
      + labels                    = (known after apply)
      + name                      = "supplychain"
      + wait_for_cluster          = false
    }

  # rancher2_project_role_template_binding.portaltruppe["supplychain"] will be created
  + resource "rancher2_project_role_template_binding" "portaltruppe" {
      + annotations        = (known after apply)
      + group_principal_id = "activedirectory_group://CN=portaltruppe,OU=_Alte_Gruppen,OU=Verwaltung,OU=domain,DC=subdomain,DC=domain,DC=de"
      + id                 = (known after apply)
      + labels             = (known after apply)
      + name               = "portalberechtigungen"
      + project_id         = (known after apply)
      + role_template_id   = "project-member"
      + user_principal_id  = (known after apply)
    }

  # rancher2_registry.expertreg["supplychain"] will be created
  + resource "rancher2_registry" "expertreg" {
      + annotations = (known after apply)
      + description = "expert Docker Registry"
      + id          = (known after apply)
      + labels      = (known after apply)
      + name        = "expertreg"
      + project_id  = (known after apply)

      + registries {
          + address  = "srv-nexus-docker-registry.subdomain.domain.de"
          + password = (sensitive value)
          + username = "k8s"
        }
    }

Plan: 5 to add, 2 to change, 0 to destroy.
```

The plugin crashes with this error

```
Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

rancher2_project.projekte["supplychain"]: Creating...
rancher2_project.projekte["portal"]: Modifying... [id=c-g7hnt:p-rnpg9]
rancher2_project.projekte["neo"]: Modifying... [id=c-g7hnt:p-6srjh]
vault_kubernetes_auth_backend_config.k8s_auth_backend_config: Modifying... [id=auth/k8s_zentrale-dev/config]
vault_kubernetes_auth_backend_config.k8s_auth_backend_config: Modifications complete after 0s [id=auth/k8s_zentrale-dev/config]
╷
│ Error: Plugin did not respond
│
│   with rancher2_project.projekte["supplychain"],
│   on rancher.tf line 60, in resource "rancher2_project" "projekte":
│   60: resource "rancher2_project" "projekte" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with rancher2_project.projekte["portal"],
│   on rancher.tf line 60, in resource "rancher2_project" "projekte":
│   60: resource "rancher2_project" "projekte" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with rancher2_project.projekte["neo"],
│   on rancher.tf line 60, in resource "rancher2_project" "projekte":
│   60: resource "rancher2_project" "projekte" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-rancher2_v1.13.0 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x1963a21]

goroutine 87 [running]:
github.com/terraform-providers/terraform-provider-rancher2/rancher2.resourceRancher2ProjectUpdate(0xc0002d2000, 0x1f4c640, 0xc00012e300, 0x0, 0x0)
        /go/src/github.com/rancher/terraform-provider-rancher2/rancher2/resource_rancher2_project.go:242 +0xc11
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).Apply(0xc000ccbef0, 0xc000c88370, 0xc0006cd7a0, 0x1f4c640, 0xc00012e300, 0x1c65b01, 0xc000e14418, 0xc000e12390)
        /go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.14.0/helper/schema/resource.go:316 +0x263
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Provider).Apply(0xc0002f4100, 0xc000d29a10, 0xc000c88370, 0xc0006cd7a0, 0xc000d0b4b8, 0xc00062e201, 0x1c68ae0)
        /go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.14.0/helper/schema/provider.go:294 +0x99
github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ApplyResourceChange(0xc00021e388, 0x23c7960, 0xc000c84000, 0xc0002b8000, 0xc00021e388, 0xc000c84000, 0xc00066db78)
        /go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.14.0/internal/helper/plugin/grpc_provider.go:885 +0x8b4
github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0x1e8f7a0, 0xc00021e388, 0x23c7960, 0xc000c84000, 0xc0003f8060, 0x0, 0x23c7960, 0xc000c84000, 0xc000c80000, 0xc03)
        /go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.14.0/internal/tfplugin5/tfplugin5.pb.go:3305 +0x217
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000683080, 0x23e1660, 0xc000683e00, 0xc0001f8800, 0xc00095d380, 0x3411740, 0x0, 0x0, 0x0)
        /go/pkg/mod/google.golang.org/grpc@v1.27.1/server.go:1024 +0x501
google.golang.org/grpc.(*Server).handleStream(0xc000683080, 0x23e1660, 0xc000683e00, 0xc0001f8800, 0x0)
        /go/pkg/mod/google.golang.org/grpc@v1.27.1/server.go:1313 +0xd3f
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc0005d5010, 0xc000683080, 0x23e1660, 0xc000683e00, 0xc0001f8800)
        /go/pkg/mod/google.golang.org/grpc@v1.27.1/server.go:722 +0xa1
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /go/pkg/mod/google.golang.org/grpc@v1.27.1/server.go:720 +0xa1

Error: The terraform-provider-rancher2_v1.13.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -20,6 +20,7 @@ BUG FIXES:
 
 * Fix `rancher2_node_pool` resource, adding `forcenew` property to not updatable arguments
 * Fix `rancher2_cluster` resource, fixing provider crash if `cluster_monitoring_input` argument is deleted
+* Fix `rancher2_project` resource, fixing provider crash if `project_monitoring_input` argument is deleted
 * Fix `rancher2_catalog_v2` resource, just setting default `git_branch` value if `git_repo` is specified
 * Fix `rancher2_cluster.eks_config_v2` argument, setting `private_access`, `public_access` and `secrets_encryption` as computed argument, removing default value
 

--- a/rancher2/resource_rancher2_app_v2_test.go
+++ b/rancher2/resource_rancher2_app_v2_test.go
@@ -22,36 +22,18 @@ var (
 
 func init() {
 	values := v3.MapStringInterface{
-		"alertmanager": map[string]interface{}{
+		"alerts": map[string]interface{}{
 			"alertmanagerSpec": map[string]interface{}{
-				"enabled":           false,
-				"useExistingSecret": true,
-				"configSecret":      "alertmanager-rancher-monitoring-alertmanager",
-			},
-		},
-		"prometheus": map[string]interface{}{
-			"prometheusSpec": map[string]interface{}{
-				"requests": map[string]interface{}{
-					"cpu":    "150m",
-					"memory": "250Mi",
-				},
+				"enabled":  false,
+				"severity": "warning",
 			},
 		},
 	}
 	valuesUpdated := v3.MapStringInterface{
-		"alertmanager": map[string]interface{}{
+		"alerts": map[string]interface{}{
 			"alertmanagerSpec": map[string]interface{}{
-				"enabled":           false,
-				"useExistingSecret": true,
-				"configSecret":      "alertmanager-rancher-monitoring-alertmanager",
-			},
-		},
-		"prometheus": map[string]interface{}{
-			"prometheusSpec": map[string]interface{}{
-				"requests": map[string]interface{}{
-					"cpu":    "150m",
-					"memory": "250Mi",
-				},
+				"enabled":  false,
+				"severity": "info",
 			},
 		},
 	}
@@ -66,10 +48,10 @@ func init() {
 	testAccRancher2AppV2 = `
 resource "` + testAccRancher2AppV2Type + `" "foo" {
   cluster_id = rancher2_cluster_sync.testacc.cluster_id
-  name = "rancher-monitoring"
-  namespace = "cattle-monitoring-system"
+  name = "rancher-cis-benchmark"
+  namespace = "cis-operator-system"
   repo_name = "rancher-charts"
-  chart_name = "rancher-monitoring"
+  chart_name = "rancher-cis-benchmark"
   values = <<EOF
 ` + valuesStr + `
 EOF
@@ -78,10 +60,10 @@ EOF
 	testAccRancher2AppV2Update = `
 resource "` + testAccRancher2AppV2Type + `" "foo" {
   cluster_id = rancher2_cluster_sync.testacc.cluster_id
-  name = "rancher-monitoring"
-  namespace = "cattle-monitoring-system"
+  name = "rancher-cis-benchmark"
+  namespace = "cis-operator-system"
   repo_name = "rancher-charts"
-  chart_name = "rancher-monitoring"
+  chart_name = "rancher-cis-benchmark"
   values = <<EOF
 ` + valuesStrUpdated + `
 EOF
@@ -103,9 +85,9 @@ func TestAccRancher2AppV2_basic(t *testing.T) {
 				Config: testAccRancher2AppV2Config,
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckRancher2AppV2Exists(testAccRancher2AppV2Type+".foo", app),
-					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "name", "rancher-monitoring"),
-					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "namespace", "cattle-monitoring-system"),
-					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "chart_name", "rancher-monitoring"),
+					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "name", "rancher-cis-benchmark"),
+					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "namespace", "cis-operator-system"),
+					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "chart_name", "rancher-cis-benchmark"),
 					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "cluster_id", testAccRancher2ClusterID),
 				),
 			},
@@ -113,9 +95,9 @@ func TestAccRancher2AppV2_basic(t *testing.T) {
 				Config: testAccRancher2AppV2UpdateConfig,
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckRancher2AppV2Exists(testAccRancher2AppV2Type+".foo", app),
-					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "name", "rancher-monitoring"),
-					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "namespace", "cattle-monitoring-system"),
-					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "chart_name", "rancher-monitoring"),
+					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "name", "rancher-cis-benchmark"),
+					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "namespace", "cis-operator-system"),
+					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "chart_name", "rancher-cis-benchmark"),
 					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "cluster_id", testAccRancher2ClusterID),
 				),
 			},
@@ -123,9 +105,9 @@ func TestAccRancher2AppV2_basic(t *testing.T) {
 				Config: testAccRancher2AppV2Config,
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckRancher2AppV2Exists(testAccRancher2AppV2Type+".foo", app),
-					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "name", "rancher-monitoring"),
-					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "namespace", "cattle-monitoring-system"),
-					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "chart_name", "rancher-monitoring"),
+					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "name", "rancher-cis-benchmark"),
+					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "namespace", "cis-operator-system"),
+					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "chart_name", "rancher-cis-benchmark"),
 					resource.TestCheckResourceAttr(testAccRancher2AppV2Type+".foo", "cluster_id", testAccRancher2ClusterID),
 				),
 			},

--- a/rancher2/resource_rancher2_project.go
+++ b/rancher2/resource_rancher2_project.go
@@ -216,20 +216,27 @@ func resourceRancher2ProjectUpdate(d *schema.ResourceData, meta interface{}) err
 					old, new := d.GetChange("project_monitoring_input")
 					oldInput := old.([]interface{})
 					oldInputLen := len(oldInput)
-					newInput := new.([]interface{})
-					newInputLen := len(newInput)
-					monitorVersionChanged = (oldInputLen != newInputLen)
-					if newInputLen > 0 && !monitorVersionChanged {
+					oldVersion := ""
+					if oldInputLen > 0 {
 						oldRow, oldOK := oldInput[0].(map[string]interface{})
-						newRow, newOK := newInput[0].(map[string]interface{})
-						if oldOK && newOK {
-							if oldRow["version"] != newRow["version"] {
-								monitorVersionChanged = true
-							}
+						if oldOK {
+							oldVersion = oldRow["version"].(string)
 						}
 					}
+					newInput := new.([]interface{})
+					newInputLen := len(newInput)
+					newVersion := ""
+					if newInputLen > 0 {
+						newRow, newOK := newInput[0].(map[string]interface{})
+						if newOK {
+							newVersion = newRow["version"].(string)
+						}
+					}
+					if oldVersion != newVersion {
+						monitorVersionChanged = true
+					}
 				}
-				if monitorVersionChanged {
+				if monitorVersionChanged && monitoringInput != nil {
 					err = updateProjectMonitoringApps(meta, newProject.ID, monitoringInput.Version)
 					if err != nil {
 						return err

--- a/scripts/start_rancher.sh
+++ b/scripts/start_rancher.sh
@@ -22,7 +22,8 @@ CERTMANAGER_NS=${CERTMANAGER_NS:-"cert-manager"}
 ## rancher
 RANCHER_VERSION=${RANCHER_VERSION:-"v2.5.6"}
 RANCHER_NS=${RANCHER_NS:-"cattle-system"}
-export RANCHER_HOSTNAME="rancher.${K3S_SERVER_IP}.xip.io"
+RANCHER_DNS_DOMAIN="nip.io"
+export RANCHER_HOSTNAME="rancher.${K3S_SERVER_IP}.${RANCHER_DNS_DOMAIN}"
 export RANCHER_IP=${K3S_SERVER_IP}
 
 # Installing helm charts 
@@ -110,5 +111,5 @@ export RANCHER_INSECURE=true
 export RANCHER_URL="https://${RANCHER_HOSTNAME}"
 export RANCHER_VERSION=${RANCHER_VERSION}
 if [ ${EXPOSE_HOST_PORTS} == "true" ]; then 
-  export RANCHER_EXPOSED_URL="https://rancher.127.0.0.1.xip.io:${K3S_INGRESS_PORT_TLS}"
+  export RANCHER_EXPOSED_URL="https://rancher.127.0.0.1.${RANCHER_DNS_DOMAIN}:${K3S_INGRESS_PORT_TLS}"
 fi

